nebula/ui/components/VPNControllerView.qml

```diff
@@ -594,18 +591,27 @@ Item {
             horizontalCenterOffset: 0
             horizontalCenter: parent.horizontalCenter
         }
-        enabled: !connectionInfoScreenVisible && !ipInfoPanel.isOpen
+        enabled: !connectionInfoScreenVisible && !ipInfoPanel.visible
 
         Accessible.ignored: connectionInfoScreenVisible || ipInfoPanel.isOpen
     }
 
     VPNIPInfoPanel {
+        property var connectionStabilityWatcher: VPNConnectionHealth.stability
+        property var vpnStateWatcher: VPNController.state
```

Comment:
Is there any benefit to creating new properties / signal handlers instead of using `Connection` objects? If not, I think the latter is more intuitive

Reply:
great point, i agree. updated.